gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -275,7 +278,7 @@ DagStateStore createDagStateStore(Config config, Map<URI, TopologySpec> topology
   }
 
   // Initializes and returns an array of Queue of size numThreads
-  private static LinkedBlockingDeque<?>[] initializeDagQueue(int numThreads) {
+  static LinkedBlockingDeque<?>[] initializeDagQueue(int numThreads) {
```

Comment:
why package protected?  is there a clear reason not to use `public`?

Reply:
initializing dag queue is not something that a totally unrelated code would require to use.